pyproject.toml

```diff
@@ -20,15 +19,22 @@ dependencies = [
     "tenacity>=9.0.0",
     "python-dateutil>=2.9.0.post0",
     "openai>=1.61.1",
+    "segment-analytics-python>=2.3.4",
+    "py-machineid>=0.7.0",
+    "httpx>=0.28.1",
 ]
 requires-python = ">=3.10"
 readme = "README.md"
 license = {text = "MIT"}
+dynamic = ["version"]
```

Comment:
What is this?

Reply:
It defines that we use the version from the build backend which is hatch in our case which in turn is configured in its corresponding section to look at `src/askui/__init__.py`